backend/utilities/debug_logging.py

```
@@ -13,8 +13,11 @@ def is_development() -> bool:
     return os.getenv('ENVIRONMENT', 'development').lower() in ['development', 'dev', 'local']
 
 def debug_log(message: str, *args, **kwargs):
-    """Debug log that only outputs in development mode"""
-    if is_development():
+    """Debug log that outputs in development mode or when DEBUG_LOGGING is enabled"""
+    environment = os.getenv('ENVIRONMENT', 'development')
```

Ruff (0.13.3)

Local variable `environment` is assigned to but never used

Remove assignment to unused variable `environment`

(F841)